app/javascript/modules/listings/BuyHeader.tsx

```diff
@@ -3,18 +3,31 @@ import React from "react"
 import Link from "../../navigation/Link"
 import "./BuyHeader.scss"
 import { BeforeApplyingForSale, BeforeApplyingType } from "../../components/BeforeApplyingForSale"
+import { renderInlineMarkup } from "../../util/languageUtil"
 
 const GetHelp = () => (
   <div className="md:bg-white md:p-4">
     <Heading styleType="underlineWeighted" className="mb-5" priority={2}>
-      {t("saleDirectory.getHelp.title")}
+      {t("listingsForSale.getHelp.helpWithDownpayment")}
     </Heading>
-    <p className="mb-4">{t("saleDirectory.getHelp.somePeopleMayQualify")}</p>
-    <p className="mb-4">{t("saleDirectory.getHelp.otherGroupsMayQualify")}</p>
-    <p>
-      <Link external={true} href="https://sfmohcd.org/loan-programs">
-        {t("saleDirectory.getHelp.viewFullListOfPrograms")}
-      </Link>
+    <span>
+      <p className="mb-4">
+        {t("listingsForSale.getHelp.dalpProgramInfo")}{" "}
+        <Link
+          external={true}
+          target="_blank"
+          href="https://sf.gov/reports/may-2023/downpayment-assistance-loan-program-dalp"
```

Comment:
Just a thought: For next year, assuming we will also have DALP program info to display, it may be easier to extract this url into a constant like "DALP_URL". And then we can just update that variable instead of the component file.

Reply:
I like that!